### PR TITLE
Fix grey color on white background

### PR DESCRIPTION
**Pull request by brianhartsock**
Part of #105 

Initial thoughts on how to clean up our grey usage.  This isn't complete yet.   Please review and let me know if you have any concerns or ideas.

Concerns:
* 333 and 666 aren't that easy to distinguish.  See the hover state.
* Zero state title is pretty bold now.  We could make it the same color as the text if we wanted.

Table Headers

Zero State (the header seems a little dark to me now)

Loading

Subtext

Progress subtext

### CODE REVIEW

**Comment by araiford** [comment]
Here are the darker grey cloud loading icons:

normal:


retina:

**Comment by brianhartsock** [comment]
I've rebased and updated the description with the latest screenshots.  Couple things to note:

I think we should create the following issues for follow up
* Investigate Table Hover Contrast
* Color Theory Improvements
* Disabled Status Indicator Contrast (as @annwa pointed out in #105)
* Improve Color Management in code - Still lots of grey's that are hard coded, could use lighten/darken functions to reduce number of hard coded colors if that makes sense.

If I get a +1, I can rebase again to squash the commits then merge.

**Comment by mazzifer** [comment]
@brianhartsock: LGTM

File: lib/stylesheets/patterns/_list-table.scss [hunk 1]
```diff
@@ -245,17 +245,17 @@ td {
...
-  color: $grey400;
+  color: $grey500;
 }
 
 /* Loading Overlay */
 .rs-table-overlay-loading .rs-table-overlay-message {
-  color: $grey300;
+  color: $grey500;
   &:before {
     background: image-url('clouding.gif') center center no-repeat;
```
Comment (bradgignac) [inline lib/stylesheets/patterns/_list-table.scss]: Needs `background-size`.

**Comment by brianhartsock** [comment]
@bradgignac fixed

**Comment by bradgignac** [comment]
:shipit:

**Comment by bradgignac** [comment]
@brianhartsock I'm waiting on an outstanding PR from @riltsken. Once that's in, I'll cut a release.

**Pull request merged**